pyanaconda/core/util.py

```diff
@@ -216,6 +216,7 @@ def sigusr1_too_late_handler(num, frame):
     def sigusr1_preexec():
         signal.signal(signal.SIGUSR1, signal.SIG_IGN)
 
+    old_sigalrm_handler = None
     try:
         old_sigusr1_handler = signal.signal(signal.SIGUSR1, sigusr1_success_handler)
```

Comment:
I guess we should also define this `old_sigusr1_handler`?

Reply:
I wanted to do minimal changes, there is more such variables but pylint is not failing on them, I guess because they are  under if: clause.
Frankly, I do not dare to touch this ancient code much.

Reply:
Well, there is also childproc and that should be it. If you insist, I can add them.

Reply:
Agreed with Radek - better keep this in place until we hopefully throw it all away once we switch to Wayland.

Reply:
Updated

Reply:
I understand that, however, this should be pretty safe change, shouldn't it?

Reply:
Done